gensim/test/test_data/test_mmcorpus_with_index.mm

```diff
@@ -0,0 +1,30 @@
+%%MatrixMarket matrix coordinate real general
```

Comment:
Can we also add test files in `mm.gz` and `mm.bz2` formats? Just to make sure stuff continues to work.